miss_islington/status_change.py

```diff
@@ -115,13 +115,13 @@ async def check_ci_status_and_approval(
                         status = "it's a success"
                     message = f"Status check is done, and {status} {emoji}."
                     if not success:
-                        if is_automerge:
-                            participants = await util.get_gh_participants(gh, pr_number)
-                        else:
-                            original_pr_number = title_match.group("pr")
-                            participants = await util.get_gh_participants(
-                                gh, original_pr_number
-                            )
+                        # if is_automerge:
+                        participants = await util.get_gh_participants(gh, pr_number)
+                        # else:
+                        #    original_pr_number = title_match.group("pr")
+                        #    participants = await util.get_gh_participants(
+                        #        gh, original_pr_number
+                        #    )
```

Comment:
Can you elaborate a bit more on the logic here?
When is `is_automerge` true? When the `automerge` label is applied to the issue or when the backport PR was approved by a core-dev review? Or both?
And what's the difference between the participants of `pr_number` and `original_pr_number`?
IIUC now in case of failure you are pinging the participants of the backport pr -- does that (only?) include whoever merged the backport PR?

(cc @DanielNoord)

Reply:
I commented out this code because it's no longer covered by tests, i.e. now really only auto-merges should emit comments on PRs.